include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -214,11 +215,12 @@ struct layout_stride {
     // TODO @proposal-bug In the proposal, the constructor doesn't take a
```

Comment:
Would this TODO still be valid after this change?

Reply:
yeah that is still an issue in the proposal. Lets fix it there.

Reply:
https://github.com/ORNL/cpp-proposals-pub/pull/169